### PR TITLE
Enable nullable: System.Management.Automation.Language.IAstPostVisitHandler

### PR DESCRIPTION
**Pull request by powercode**
Tracking issue: #12631.

### CODE REVIEW

**Review by rjmholt — APPROVED** [review]

**Comment by rjmholt** [comment]
@PoshChan please remind me in 1 day

**Comment by PoshChan** [comment]
@rjmholt, this is the reminder you requested 1 day ago

**Comment by rjmholt** [comment]
@powercode looks like the build is failing due to changes in this PR:

```
/home/vsts/work/1/s/src/System.Management.Automation/engine/parser/PreOrderVisitor.cs(252,1): error SA1507: Code should not contain multiple blank lines in a row [/home/vsts/work/1/s/src/System.Management.Automation/System.Management.Automation.csproj]
```

File: src/System.Management.Automation/engine/parser/PreOrderVisitor.cs [hunk 1]
```diff
@@ -250,10 +250,12 @@ public abstract class AstVisitor2 : AstVisitor
         public virtual AstVisitAction VisitPipelineChain(PipelineChainAst statementChain) => DefaultVisit(statementChain);
     }
 
+
```
Comment (rjmholt) [inline src/System.Management.Automation/engine/parser/PreOrderVisitor.cs:253]:
```suggestion
```

**Comment by rjmholt** [comment]
@PoshChan please remind me in 1 hour

**Comment by PoshChan** [comment]
@rjmholt, this is the reminder you requested 1 hour ago

**Pull request merged**